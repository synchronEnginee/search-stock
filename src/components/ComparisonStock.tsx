/** @jsxImportSource @emotion/react */
import React from "react";
import { css } from "@emotion/react";

export interface ComparisonStockProps {
  name: string;
  code: string;
}

export const ComparisonStock: React.FC<ComparisonStockProps> = (props) => {
  const { name, code } = props;
  const styles = css({
    td: {
      border: "inherit",
    },
  });
  return (
    <tr css={styles}>
      <td>{name}</td>
      <td>{code}</td>
    </tr>
  );
};
