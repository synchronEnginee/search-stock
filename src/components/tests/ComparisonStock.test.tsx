import React from 'react';
import renderer from 'react-test-renderer';
import ComparisonStock from '../ComparisonStock';

// ここにテストを書いていく
// スナップショットテスト
test('スナップショットテスト', () => {
  const props = {
    code: 9999,
  };
  const tree = renderer.create(<ComparisonStock code={props.code} />);
  expect(tree).toMatchSnapshot();
});
