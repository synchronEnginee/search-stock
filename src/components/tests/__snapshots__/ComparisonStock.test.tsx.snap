// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`スナップショットテスト 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-4xrqo5 p{font-size:26px;}",
      }
    }
    data-emotion="css 4xrqo5"
  />,
  <tr
    className="css-4xrqo5"
  >
    <td>
      9999
    </td>
    <td>
      ---
    </td>
    <td>
      0
    </td>
    <td>
      0
    </td>
    <td>
      0
    </td>
    <td>
      0
    </td>
  </tr>,
]
`;
